test/WebJobs.Extensions.Tests/Extensions/NotificationHubs/NotificationHubAsyncCollectorTests.cs

```diff
@@ -2,16 +2,21 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Threading.Tasks;
 using Microsoft.Azure.NotificationHubs;
 using Microsoft.Azure.WebJobs.Extensions.NotificationHubs;
+using Microsoft.Azure.WebJobs.Extensions.Tests.Common;
 using Moq;
 using Xunit;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Tests.NotificationHubs
 {
     public class NotificationHubAsyncCollectorTests
     {
+        public static string debugLogNoResults = "TrackingId = .Results is null or empty. Refer to NotificationHubs documentaion for more details on how to debug issues while sending push notifications.";
```

Comment:
Since this is meant to be a debugging tool, let's give all the info that we can. I looked at the types and I think we can do something like this:

```
Notification Hubs Test Send
  NotificationId = notificationid
  TrackingId = trackingid
  State = Completed
  Results (0 success, 0 failure)
```
and for success
```
Notification Hubs Test Send
  NotificationId = notificationid
  TrackingId = trackingid
  State = Completed
  Results (3 success, 0 failure)
    ApplicationPlatform = Windows, RegistrationId = guid-guid-guid, Outcome = outcome
    ApplicationPlatform = Windows, RegistrationId = guid-guid-guid, Outcome = outcome
    ApplicationPlatform = Windows, RegistrationId = guid-guid-guid, Outcome = outcome
```